packages/firebase_ui_auth/lib/src/screens/profile_screen.dart

```diff
@@ -439,7 +439,7 @@ class _EmailVerificationBadgeState extends State<_EmailVerificationBadge> {
                 if (state != EmailVerificationState.sent)
                   LoadingButton(
                     isLoading: state == EmailVerificationState.sending,
-                    label: 'Send verification email',
+                    label: l.sendVerificationEmailButtonText,
```

Comment:
```suggestion
                    label: l.sendVerificationEmailLabel,
```
Do you mind updating this label to this by updating the arb files and generating the code, please? 👍

Reply:
Yes, I will fix it and push it tonight.